packages/votura-validators/src/objects/election.ts

```diff
@@ -122,3 +125,14 @@ export const updateableElectionObjectSchema = z.toJSONSchema(
   updateableElectionObject,
   toJsonSchemaParams,
 );
+
+export const freezableElectionObject = electionObject.pick({
+  freezable: true,
```

Comment:
Maybe also return the `id`? Perhaps practical that you know to which election this response belongs?
```suggestion
  id: true,
  freezable: true,
```

…but normally you remember for which election you have asked the backend…